include/h2o.h

```diff
@@ -615,7 +615,12 @@ struct st_h2o_context_t {
         /**
          * link-list of h2o_http1_conn_t
          */
-        h2o_linklist_t _conns;
+        h2o_linklist_t _active_conns;
+        /**
+         * link-list of h2o_http1_conn_t
+         */
+        h2o_linklist_t _inactive_conns;
```

Comment:
* Maybe call it `_idle_conns`?
* I think that the desirable behavior is to cull the ones that have been idle for long time, regardless of the underlying protocol. Then, IMO what we should do is either:
  * a) have a unified, protocol-agnostic linked list of connections, or
  * b) have a per-protocol linked list that stores idle connections in the descending order of time while the connections have been idle, as well, for each connection, retain when it became idle.